### PR TITLE
TSOPS-328: Support bundle improvements.

### PR DESCRIPTION
**Pull request by Roland0**
3 Commits:

1) Added getting k8s node and event data
2) Added feature to get previous container logs after restart
3) Changed descriptions from JSON to YAML

### CODE REVIEW

File: scripts/get_support_bundle.sh [hunk 1]
```diff
@@ -37,10 +42,17 @@ for object in svc deployment sts pvc daemonset ingress replicaset; do
...
     for item in ${items}; do
-        kubectl ${KUBE_OPTS} get ${object} ${item} -o json > ${LOG_DIR}/${object}/${item}-kubectl.json
+        kubectl ${KUBE_OPTS} get ${object} ${item} -o yaml > ${LOG_DIR}/${object}/${item}-kubectl.yaml
     done
 done
 
+# get node data, might be nice to have a selector here eg: -l role=sysdigcloud
+kubectl get nodes > ${LOG_DIR}/nodes.txt
+echo " " >> ${LOG_DIR}/nodes.txt
+kubectl describe nodes >> ${LOG_DIR}/nodes.txt
```
Comment (mbreitung) [inline scripts/get_support_bundle.sh]: why not just push `descibe nodes` to a different file instead of appending?

Comment (Roland0) [reply scripts/get_support_bundle.sh]: @mbreitung I thought I'd add all of the "node" data into nodes.txt

It seems to work ok, I'm open to any suggestions etc

Comment (mbreitung) [reply scripts/get_support_bundle.sh]: How about creating a single file that has the list of nodes and a separate file for each node's `describe`?

Comment (Roland0) [reply scripts/get_support_bundle.sh]: Good idea.

**Pull request closed without merging**